src/components/FileMenu/__tests__/FileMenu.spec.js

```diff
@@ -0,0 +1,340 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import FavoritesDialog from '@dhis2/d2-ui-favorites-dialog'
+import TranslationDialog from '@dhis2/d2-ui-translation-dialog'
+import SharingDialog from '@dhis2/d2-ui-sharing-dialog'
+import { Popover } from '@dhis2/ui'
+
+import { FileMenu } from '../FileMenu'
+import { RenameDialog } from '../RenameDialog'
+import { DeleteDialog } from '../DeleteDialog'
+import { GetLinkDialog } from '../GetLinkDialog'
+import { SaveAsDialog } from '../SaveAsDialog'
+
+describe('The FileMenu component ', () => {
```

Comment:
Nice test coverage. Some of these tests where you are checking what is rendered based on props could be accomplished with snapshots. Also, if I understand correctly, people are starting to favor testing-library over enzyme.  That said, the tests are good as is too, if you don't feel like doing this now.

Reply:
I will have to work on this component again when the new SharingDialog is ready, I will review the tests then 👍